cellxgene_resources/curation_qa.ipynb

```diff
@@ -438,6 +438,39 @@
     "    display(deprecated_donors)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Check if any human donors' annotated sex is inconsistent with raw expression counts**<br>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "donor_sex_df = evaluate_donors_sex(adata)\n",
+    "donor_sex_df[donor_sex_df['scRNAseq_sex'] != donor_sex_df['author_annotated_sex']]"
```

Comment:
Would be ideal to follow other behavior in the notebook - use the report() function for printing ERROR cases, and can also use that to print when everything looks GOOD.
Don't need to display the dataframe when there isn't an error as it's just empty.

Reply:
The example from the donor metadata a few cells above...
```
inconsistencies = donor_df[donor_df.duplicated(subset='donor_id', keep=False) == True].sort_values('donor_id')
if not inconsistencies.empty:
    report('donor metadata inconsistencies', 'ERROR')
    display(inconsistencies)
```